src/els_hover_provider.erl

```diff
@@ -57,6 +57,33 @@ documentation(_M, _POI) ->
 %% @doc get the docs
 -spec get_docs(atom(), atom(), byte()) -> binary().
 get_docs(M, F, A) ->
+  %% Use apply and hard coded record so it can be compiled on older OTP's
```

Comment:
We should probably be more specific on what "older" means here. I suggest we update the `edoc` for this function with a nicer description, mentioning the minimum required OTP version for this to work.